frontend/src/static/js/main.js

```diff
@@ -214,10 +215,11 @@ window.app = new window.Vue({
       const info = {
         author: hash.tabAuthor,
         repo: hash.tabRepo,
+        isMergeGroup: hash.authorshipIsMergeGroup === 'true',
```

Comment:
Might want to rename it to `isMergeAuthorshipGroup`? Since there are 2 ways of merging group.

Reply:
Hmm don't think this is necessary as `isMergeGroup` is contained in the `info` object, which will be passed to authorship tab

Reply:
Ahh you are right. Was forgetting which file I was looking at.